health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/FiltersNational.js

```
@@ -0,0 +1,110 @@
+          </span>
+        </div>
+      )}
+      {showDateRange && (
+        <div className="filters-input">
+          <DateRange onFilterChange={handleFilterChange} values={value?.range} t={t} />
+        </div>
+      )}
+      <div className="filters-input">
+        <div className="mbsm">{t(isNational ? "ES_DSS_STATE" : "ES_DSS_DDR")}</div>
+        <MultiSelectDropdown
+          options={ulbTenants?.ddr && ulbTenants.ddr?.sort((x, y) => x?.ddrKey?.localeCompare(y?.ddrKey))?.map(ele=>({...ele,i18Key:`DSS_TB_${Digit.Utils.locale.getTransformedLocale(ele?.ddrKey)}`}))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)